marlowe-playground-client/webpack.test.config.js

```diff
@@ -34,49 +31,45 @@ module.exports = {
                 test: /\.purs$/,
                 use: [
                     {
-                        loader: 'purs-loader',
+                        loader: "purs-loader",
                         options: {
                             src: [
-                                'src/**/*.purs',
-                                'generated/**/*.purs',
-                                '.spago/*/*/src/**/*.purs',
-                                'web-common/**/*.purs',
-                                'web-common-marlowe/**/*.purs',
-                                'web-common-playground/**/*.purs',
-                                'test/**/*.purs'
+                                "src/**/*.purs",
+                                "generated/**/*.purs",
+                                ".spago/*/*/src/**/*.purs",
+                                "web-common/**/*.purs",
+                                "web-common-marlowe/**/*.purs",
+                                "web-common-playground/**/*.purs",
+                                "test/**/*.purs",
                             ],
                         }
                     }
                 ]
             },
             {
-                test: /\.(gif|png|jpe?g|svg)$/i,
-                use: 'url-loader'
+                test: /\.(png|svg|jpg|jpeg|gif)$/i,
+                type: "asset/resource",
             },
         ]
     },
-
     resolve: {
         modules: [
-            // We need the second entry for node to be able to
-            // locate `node_modules` from client directory when 
-            // modules are referenced from inside `web-common`.
-            'node_modules', path.resolve(__dirname, './node_modules')
+            "node_modules",
+            // We need this entry for node to be able to locate `node_modules` from
+            // client directory when modules are referenced from inside `web-common`.
+            path.resolve(__dirname, "./node_modules"),
         ],
         alias: {
-            grammar: path.resolve(__dirname, './grammar.ne'),
-            static: path.resolve(__dirname, './static'),
-            src: path.resolve(__dirname, './src')
+            grammar: path.resolve(__dirname, "./grammar.ne"),
+            static: path.resolve(__dirname, "./static"),
+            src: path.resolve(__dirname, "./src"),
         },
-        extensions: ['.purs', '.js', '.ts']
+        extensions: [".purs", ".js", ".ts"],
```

Comment:
Drop .ts ?

Reply:
no, we need it for the Marlowe playground